recipes/nanocube/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "nanocube" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/nanocube-{{ version }}.tar.gz
+  sha256: 5737404f67c8cc1cc05c308a9fc08d04bf0e99568009dd07b1b3c904e8368863
+  patches:
+    - 0001-get-version.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  script_env:
+    - VERSION={{ version }}
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.10
+    # Pip check complains about numpy version
+    # - numpy <2,>=1.17.3
+    - pandas
+    - pyroaring
+    - pyarrow
+    - sortednp
```

Comment:
Looks like this one is the culprit for the pip-check failure:

```
sortednp 0.5.0 has requirement numpy<2,>=1.17.3, but you have numpy 2.1.2.
```

Reply:
See https://github.com/conda-forge/sortednp-feedstock/pull/17

Reply:
@ocefpaf gotcha; I'll remember to check likewise for future recipes. For this one we can have for the `sortednp` PR to be merged and hopefully it'll go away

Reply:
Just merged that. Wait a few hours to restart things here. We still may need to patch/remove some old `sortednp` but I'm clocking out for the day.